apps/bfd-shared-utils/pom.xml

```diff
@@ -16,13 +16,22 @@
 	</description>
 
 	<dependencies>
-			<dependency>
-				<!-- Just useful. -->
-				<groupId>com.google.guava</groupId>
-				<artifactId>guava</artifactId>
-				<version>24.1.1-jre</version>
-			</dependency>
-
+		<dependency>
+			<!-- Just useful. -->
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>24.1.1-jre</version>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.2</version>
+		</dependency>
+		<dependency>
+			<groupId>org.glassfish.jaxb</groupId>
+			<artifactId>jaxb-runtime</artifactId>
+			<version>2.3.1</version>
+		</dependency>
```

Comment:
do we still need these changes in here since these dependencies are now in a bunch of other places?